core/arch/arm32/plat-stm/main.c

```diff
@@ -78,6 +78,8 @@ void main_init(uint32_t nsec_entry)
 	struct sm_nsec_ctx *nsec_ctx;
 	size_t pos = get_core_pos();
 
+	asc_init();
```

Comment:
It could be that this is too early for the STM platform, @pascal-brand-st what do you think?